api/bundlephobia.ts

```diff
@@ -45,6 +51,18 @@ async function handler ({ topic, scope, name }: PathArgs) {
         status: byteSize(gzip, { units: 'iec' }).toString().replace(/iB\b/, 'B'),
         color: 'blue'
       }
+    case 'dependency-count':
+      return {
+        subject: 'dependency count',
+        status: dependencyCount,
+        color: 'blue'
+      }
+    case 'tree-shaking':
+      return {
+        subject: 'tree-shaking',
```

Comment:
Seems "tree shaking" is already a well known term https://en.wikipedia.org/wiki/Tree_shaking
We can call it `tree shaking` now :D
```suggestion
        subject: 'tree shaking',
```

Reply:
Yeah, you're right. Webpack docs use the same name:
https://webpack.js.org/guides/tree-shaking/